packaging/el8/cloud.spec

```diff
@@ -55,7 +55,7 @@ intelligent IaaS cloud implementation.
 
 %package management
 Summary:   CloudStack management server UI
-Requires: java-17-openjdk
+Requires: (java-17-openjdk or java-21-openjdk)
```

Comment:
@vishesh92 
is JDK21 support part of  EL10 support or a separated PR ?

cc @rohityadavcloud

Reply:
The default version of JDK present on the EL10 is 21. I am still testing, but I don't see any issues as of now to run packages with java 21.